apps/sms/views/inbox/js/inbox.js

```diff
@@ -2,9 +2,9 @@
 /* vim: set shiftwidth=2 tabstop=2 autoindent cindent expandtab: */
 
 /*global Template, Utils, Threads, Contacts, Threads,
-         WaitingScreen, MessageManager, TimeHeaders,
```

Comment:
I mean, we set  this.list/this.listData to TimeHeader and updataAll function could handle all the related string updating, including the fast list. But this behavior will be changed once we're in the split mode and each view could have different way to schedule the string update.

>  I doubt our app will be always visible and we should strive to real-time accuracy
I would be glad that we can have a smarter way to update the time related string instead of struggling in some edge cases. We already update the strings when visibility change, maybe we can just set the actual(but it's questionable that how accurate will it be if the duration is really long and app is always on the foreground that without resetting the timeout) timeout for the next necessary changes, not update the by every 10 sec.